.github/DEVELOPMENT.md

```diff
@@ -74,6 +74,42 @@ Your command line application will then display some information about Eleventy,
 
 You can also use the `External` URL to preview the local copy on another device connected to the same network, which helps you check to see how the site looks and functions on things like smartphones. This is done via [Browsersync](https://www.browsersync.io/).
 
+### Different npm scripts
+
+This is a list of all the **npm** commands you could run from the command line within this project.
+
+#### Building the site
+
+The `npm run build` command will build the site into the `_site` folder. It is used for the production server hosted with Github pages.
+
+#### Running in development mode
+
+The `npm run dev` command will build the site into the `_site` folder and set the `ELEVENTY_ENV=dev` variable for the Netlify staging instance of the site.
+
+#### Building the HTML files
+
+The `npm run eleventy` command builds the HTML pages from the `src` folder into the `_site`.
+
+#### Creating a new folder and the files needed
+
+To create a new folder for the site in both the `src/en` and `src/fr` directories along with the basic files needed for a page in these folders you can use `npm run new-folder`.
+
+The script will need the name of the folder, the English title of the page, the French title of the page, the English description of the page, the French description of the page.
+
+The script will create the json file for the breadcrumbs, an index.md, and the english and french folders.
+
+#### Checking the spelling on the site
+
+The `npm run spellcheck` command runs a spell checker on all the markdown and HTML files inside the `src` folder.
+
+The script will then display possible spelling errors in the terminal.
+
+#### Bulding the site for testing locally with a server
```

Comment:
```suggestion
#### Bulding the site for testing locally on port 8081
```

Testing it locally imply a server and specifying the port make it more implicit.

Reply:
Just removed server and not putting a port because it's not always 8081